man/networkd.conf.xml

```diff
@@ -414,6 +414,117 @@ DUIDRawData=00:00:ab:11:f9:2a:c2:77:29:f9:5c:00</programlisting>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[ModemManager] Section Options</title>
```

Comment:
You added documents at wrong place.
Please move this to systemd.network.xml.
This is for networkd.conf, which controls global behavior.